src/library/scala/runtime/RichByte.scala

```diff
@@ -27,7 +26,10 @@ final class RichByte(val self: Byte) extends AnyVal with ScalaWholeNumberProxy[B
 
   override def isValidByte   = true
 
+  // These are all redefined to avoid VC boxing (they also avoid the indirection through `num`)
   override def abs: Byte             = math.abs(self).toByte
   override def max(that: Byte): Byte = math.max(self, that).toByte
   override def min(that: Byte): Byte = math.min(self, that).toByte
+  override def signum: Int           = math.signum(self.toInt)
```

Comment:
That's not binary compatible, or is it? Doesn't this create a new method `RichByte$.signum$extension`?

Reply:
That's correct, which is why the next commit removes it.

The reason this commit is green is because the commit predates the fix in #9459.  If I rebase it it will break, and then I'll have to either temporarily exclude the problem or squash the commits.

I'll probably have to do something anyways, either by removing the commented out code, or keeping it with a comment.  WDYT we should do?

Reply:
👍 thanks! Glad that #9459 didn't uncover anything unexpected.

For the commented out methods, I'd probably delete them, but I'm fine either way.